### ISSUE TITLE
Empty space between the first tab & the topbar

### ISSUE THREAD

**Issue by ridwanpr** [zen-browser/desktop#2779]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Somehow after updating to the latest version there are large gap between the first tab and the top bar.
![image](https://github.com/user-attachments/assets/f1824fcc-2ec5-4ab6-98cd-bdbc5327f34e)

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.18

### What platform are you seeing the problem on?

Windows

**Comment by georgeat8** [comment]
Seems to be caused by the newly added workspace title in the top part. There are two scenarios:

- If workspaces feature is enabled, expanded sidebar will show the title, but the unexpended sidebar will have just the padding from the top.
- If workspaces feature is disabled, expanded/unexpanded sidebar will have just the padding from the top

**Comment by ridwanpr** [comment]
@georgeat8 Yeah, can confirm. It because `zen.workspace.enabled` is set to false. So zen will show empty space instead of the text `Default Workspace`

**Comment by fxzxmicah** [comment]
Still exists on ver a19.

**Comment by thechief77** [comment]
> Still exists on ver a19.

yeah didn't fix it yet. Like @ridwanpr said it because of Default Workspace.

**Comment by rubaboo** [comment]
Same. If workspaces feature is disabled, the empty spot should not be there for sure. Even when enabled, displaying the workspace title should be optional, since there are other ways the workspaces can be distinguished.

![image](https://github.com/user-attachments/assets/9bf90824-1900-4abc-b36b-2f177f5ba420)

**Comment by Smooth-E** [comment]
Possible duplicate of #2862? If this issue is present after an update for you, I suggest resetting the layout to default state.

Edit: didn't consider this one was actually opened earlier than 2862

**Comment by rubaboo** [comment]
Resetting layout did not help. This `userChrome.css` workaround hides the empty space:
```
#zen-current-workspace-indicator:has(> #zen-current-workspace-indicator-name:empty) {
  display: none !important;
}
```

**Comment by thechief77** [comment]
> Resetting layout did not help. This `userChrome.css` workaround hides the empty space:
> 
> ```
> #zen-current-workspace-indicator:has(> #zen-current-workspace-indicator-name:empty) {
>   display: none !important;
> }
> ```

that works pretty well, appreciate it mate.

**Comment by thechief77** [comment]
With last update it just solved today at the time I'm texting that comment.

**Comment by fxzxmicah** [comment]
Yes, I can confirm that.

**Comment by JSGRD22** [comment]
Option to disable workspaces is removed in 1.6b and this problem should be resolved
